common/actions.cpp

```diff
@@ -6219,8 +6219,8 @@ const char *action_ui_name_default(int act)
     // TRANS: Rec_ycle Unit (100% chance of success).
     return N_("Rec%sycle Unit%s");
   case ACTION_DISBAND_UNIT:
-    // TRANS: _You're Fired (100% chance of success).
-    return N_("%sYou're Fired%s");
+    // TRANS: _Disband Unit (100% chance of success).
+    return N_("%sDisband Unit%s");
```

Comment:
The reason I'm asking is this change to `actions.cpp`. So this won't effect rulesets that use alternate language? Do we need to keep both in the `cpp` file?

Reply:
I see, good call! This is used as a default when no action name is specified in the ruleset and to avoid saving the default names in the ruleset editor.